tests/Makefile.am

```diff
@@ -21,19 +21,15 @@
 #  SOFTWARE.
 #
 
-bin_PROGRAMS = testlayers
+bin_PROGRAMS = testlayers colorcorrection
 
 testlayers_LDFLAGS = \
 	-no-undefined
 
 AM_CPP_INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/public -I../common/core -I../common/utils -I../common/compositor -I../common/display  -I../os/linux -I./common -I./third_party/json-c
-AM_CPPFLAGS = -std=c++11 -fPIC -O2 -D_FORTIFY_SOURCE=2 -fstack-protector-strong -fPIE
+AM_CPPFLAGS = -std=c++11 -DUSE_MINIGBM -fPIC -O2 -D_FORTIFY_SOURCE=2 -fstack-protector-strong -fPIE
```

Comment:
hmm u shoul remove this since we can compile with minigbm or without it. Also below changes related to Minigbm, shouldnt affect default builds